src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
According to the checklist, it's recommended to be consistent with your vertical margins. It's better to add only top or only bottom margin, but not both. Here you have added both top and bottom margins to the `header` class. Consider removing one of them to improve consistency.